module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp

```diff
@@ -57,15 +57,15 @@
 #if defined(EINK_ROTATE_90_CLOCKWISE)
 #define EINK_DISPLAY_RES_X         (BOARD_EINK_DISPLAY_RES_Y)
 #define EINK_DISPLAY_RES_Y         (BOARD_EINK_DISPLAY_RES_X)
-#define EINK_DISPLAY_X_AXIS        (BOARD_EINK_DISPLAY_RES_Y - Y - H)
-#define EINK_DISPLAY_Y_AXIS        (BOARD_EINK_DISPLAY_RES_X - X - W)
+#define EINK_DISPLAY_X_AXIS        (BOARD_EINK_DISPLAY_RES_Y - frame.pos_y - frame.height)
+#define EINK_DISPLAY_Y_AXIS        (BOARD_EINK_DISPLAY_RES_X - frame.pos_x - frame.width)
```

Comment:
It limits use of the `EINK_DISPLAY_X_AXIS` to the scope where `frame` is limited. I would replace it with function taking necessary parameters or at least add parameter to the define. Function would be much better tho.

Reply:
check it out now :)